src/legacy/core_plugins/kibana/public/home/tutorial_resources/logos/cisco.svg

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
❤️ @sayden! fyi I was encouraged to minify svgs in one of my PRs in past: https://github.com/elastic/kibana/pull/54102#discussion_r364707276

Reply:
Oh, good to know!

Reply:
All icons minified!